downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -16,7 +16,24 @@
 
 . Under *Provided APIs*, locate the {EDAName} modal and click *Create instance*. 
 +
-This takes you to the Form View to customize your installation.
+This takes you to the Form View to customize your installation. 
++
+[IMPORTANT]
+====
+To ensure that you can run concurrent {EDAName} activations efficiently, you must set your maximum number of activations to 12 in the YAML view. 
```

Comment:
The default value if the user does not specify anything is 12.  The wording here seems to imply that if they don't set it they are going to be in trouble.  Could we reword this to say something like:
```suggestion
To ensure that you can run concurrent {EDAName} activations efficiently, set your maximum number of activations proportional to the resources available on your cluster. You can do this by the {EDAName} settings in the YAML view. 
```

Do we have an idea of roughly how much memory/RAM each actively running rulebook activation uses?  It would be great to say something to the effect of:

> Each rulebook activation uses roughly 200Mi of memory. So if you reserve 2Gi of memory for your rulebook activations, you can expect to be about to run 8 concurrent rulebook activations. These are rough approximations for planning purposes, keep in mind that this will depend on your hardware and the tasks in your rulebook. 

These are questions we should ask Ben, Alejandro, and Elijah in the #eda-technical-discussion channel.

Reply:
@rooftopcellist is right. I think there is a misunderstanding because the RHEL installer applies a default value of 12. That is the number recomended by me based on the hardware recommended for VM's. 

IMO, the goal of the documentation is to advise that number might be changed and how to do it. 
I agree that we should provide some guideline about how to scale the number of activations. 
I propose the following text:

```
When activating a rulebook under standard conditions, it typically utilizes approximately 250MB of memory. However, the actual memory consumption can vary significantly based on the complexity of your rules and the volume and size of the events processed. In scenarios where a large number of events are anticipated or the rulebook complexity is high, we strongly advise conducting a preliminary assessment of resource usage in a staging environment. This step will enable you to accurately determine the necessary number of rulebook activations. 
```

Reply:
Thanks, @Alex-Izquierdo and @rooftopcellist. I will update the content according to your suggestions and submit the PR again.